libs/components/autonumeric/src/lib/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -116,6 +111,19 @@ export class SkyAutonumericDirective
 
         this.#changeDetector.markForCheck();
       });
+
+    // On Android browsers, the native `input` event fires before AutoNumeric's
+    // `autoNumeric:rawValueModified` event (on other browsers, the order is reversed).
+    // To support Android without breaking existing behavior, we listen to both events.
+    // See: https://github.com/autoNumeric/autoNumeric/issues/781
+    fromEvent(this.#elementRef.nativeElement, 'autoNumeric:rawValueModified')
+      .pipe(takeUntil(this.#ngUnsubscribe))
+      .subscribe(() => {
+        // Prevent processing during programmatic changes via writeValue().
+        if (!this.#isWritingValue) {
```

Comment:
Any way that we could do this deterministic? Where we know that the write value would call this once? Maybe if we listen to this event in the `writeValue` with a subscription with a `takeFirst` that then sets `isWritingValue` back to false? The current logic seems to introduce a possibility that this event would not fire prior to the end of the `writeValue` function that sets the `isWritingValue` back to false.